src/devtools/client/debugger/src/actions/breakpoints/index.js

```diff
@@ -158,8 +158,8 @@ export function runAnalysisOnLine(cx, line) {
 
     const options = { logValue: "dummyValue" };
     const location = {
-      sourceId: source.id,
-      sourceUrl: source.url,
+      sourceId: source?.id,
+      sourceUrl: source?.url,
```

Comment:
so... we have this `if (!source)` above. Could you look into why this might be happening? perhaps there is some metadata about the recording and source in sentry / logrocket...

Reply:
i feel dumb for not catching that, i'll dig farther